XCUITests/ReaderViewUITest.swift

```diff
@@ -35,15 +35,20 @@ class ReaderViewTest: BaseTestCase {
 
     // Smoketest
     func testAddToReadingList() {
-        // Initially reading list is empty
+        // Navigate to reading list
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        navigator.goto(BrowserTabMenu)
         navigator.goto(LibraryPanel_ReadingList)
-
-        // Check the button is selected (is disabled and the rest bookmarks and so are enabled)
-        XCTAssertFalse(app.buttons["LibraryPanels.ReadingList"].isEnabled)
+        
+        // Check that ReadingList and Bookmarks button is enabled
+        XCTAssertTrue(app.buttons["LibraryPanels.ReadingList"].isEnabled)
```

Comment:
The only change to have this test passing again is modify these two lines with:
```
XCTAssertFalse(app.buttons["LibraryPanels.Bookmarks"].isSelected)
XCTAssertTrue(app.buttons["LibraryPanels.ReadingList"].isSelected)
```
That's to verify that when being in the ReadingList panel the blue icon at the bottom is selected. 
With this change, and only this, the test works